app/scienceapi/events/models.py

```diff
@@ -70,15 +72,26 @@ class Event(models.Model):
         help_text='Slug appended to the Mozilla Science event\'s '
                   'url that represents this event',
     )
+    is_virtual = models.BooleanField(
+        default=False,
+        help_text='Check this box if your event is happening virtually, '
+                  'that is it\'s not tied to a particular location.'
+    )
+
+    ALL_TIMEZONES = [(i, i) for i in pytz.all_timezones]
+    timezone = models.CharField(
+        max_length=40,
+        choices=ALL_TIMEZONES,
+        help_text='Timezone of the place where the event will be held. '
+                  'If the event is virtual select any timezone you like. '
+                  'It is required that you input all datetimes in this '
+                  'timezone.'
+    )
+
     date_created = models.DateTimeField(auto_now_add=True)
     date_updated = models.DateTimeField(auto_now=True)
     starts_at = models.DateTimeField()
```

Comment:
It definitely stores an offset (and Postgres definitely stores it as well at the db layer). But since we're storing it in UTC time, the offset is 0. By default, you should be getting the dates formatted as per the ISO 8601 spec. That means that there should be timezone info in the serialized field but it will show up as `.000Z` since it will be at 0 offset from UTC.